test/test_nn.py

```diff
@@ -4918,6 +4918,7 @@ def test_pdist_empty_row(self):
             inp = torch.randn(1, 3, dtype=torch.double, device=device, requires_grad=True)
             self.assertTrue(gradcheck(F.pdist, (inp,)))
 
+    @skipIfRocm
```

Comment:
Why are we now skipping this?

Reply:
it currently fails with the latest compiler changes.

Reply:
Interesting, I hadn't seen that. Let's see if the CI also confirms this once it's again up...